lib/s3middleware/lifecycleHelpers/LifecycleUtils.ts

```diff
@@ -239,6 +316,16 @@ export default class LifecycleUtils {
                     currentDate,
                 });
             }
+
+            const hasNoncurrentVersionTransitions = Array.isArray(rule.NoncurrentVersionTransitions) && rule.NoncurrentVersionTransitions.length > 0;
+            if (hasNoncurrentVersionTransitions && this._supportedRules.includes('noncurrentVersionExpiration')) {
```

Comment:
should the supported rule be `noncurrentVersionTransition` ?

Reply:
Yes, thanks!